src/setup-beam.js

```diff
@@ -184,7 +184,10 @@ async function getOTPVersions(osVersion) {
     .forEach((line) => {
       const otpMatch = line.match(/^(OTP-)?([^ ]+)/)
 
-      const otpVersion = otpMatch[2]
+      let otpVersion = otpMatch[2]
+      if (semver.validRange(otpVersion)) {
```

Comment:
@starbelly, @ericmj, @wojtekmach, could I ask your 👀 on this so we can merge it and ship 1.7.1? I'd want to pull this to a couple of repo.s, already. Thanks much.

If you have a better, even less verbose for example, "fix", do tell. I'd like to potentially consider dropping the ver-ranges, but not in this pull request.